lib/decompress_impl.ml

```diff
@@ -500,8 +500,10 @@ module RFC1951_deflate = struct
     ; i_len: int
     ; level: int
     ; wbits: int
+    ; read: int32
     ; write: int
     ; adler: Checkseum.Adler32.t
+    ; crc: Checkseum.Crc32.t
```

Comment:
We should avoid a doublon between `adler` and `crc`. This question should be hard when it can have an impact on performances.